CI Failing to build AH stack

<!--- Verify first that your issue is not already reported on GitHub -->
<!--- Also test if the latest release are affected -->

# Summary
CI currently failing on 'build stack' step

# Issue Type

- Bug Report

# Ansible, Collection, Private Automation Hub details
All versions
<!--- Paste verbatim output between triple backticks -->


# OS / ENVIRONMENT
CI - GH Actions
<!--- Provide all relevant information below, e.g. target OS versions, network device firmware, etc. -->

# Desired Behavior
Stack builds
<!--- Describe what you expected to happen when running the steps above -->

# Actual Behavior

```
Run make docker/build
INFO: Using compose profile standalone
INFO: galaxy_ng packages installed from source
INFO: Image suffix is unset
INFO: Volume suffix is unset
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.[8](https://github.com/ansible/galaxy_collection/actions/runs/7532803957/job/20504121587#step:16:9).18/x64/bin/docker-compose", line 8, in <module>
    sys.exit(main())
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/main.py", line 81, in main
    command_func()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/main.py", line 200, in perform_command
    project = project_from_options('.', options)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/command.py", line 60, in project_from_options
    return get_project(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/command.py", line 152, in get_project
    client = get_client(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/docker_client.py", line 41, in get_client
    client = docker_client(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/compose/cli/docker_client.py", line [12](https://github.com/ansible/galaxy_collection/actions/runs/7532803957/job/20504121587#step:16:13)4, in docker_client
    kwargs = kwargs_from_env(environment=environment, ssl_version=tls_version)
TypeError: kwargs_from_env() got an unexpected keyword argument 'ssl_version'
make: *** [Makefile:83: docker/build] Error 1
Error: Process completed with exit code 2.
```

--- a/.github/workflows/ci_standalone_versioned.yml
+++ b/.github/workflows/ci_standalone_versioned.yml
@@ -55,7 +55,10 @@ jobs:
         run: sudo apt-get install -y libsasl2-dev python3-dev libldap2-dev libssl-dev build-essential
 
       - name: Install docker-compose
-        run: pip3 install --upgrade docker-py docker-compose
+        run: pip3 install --upgrade docker-compose
+
+      - name: Downgrade docker
+        run: pip3 install docker==6.1.3
 
       - name: collect system info
         run: whoami; id; pwd; ls -al; uname -a ; df -h .; mount ; cat /etc/issue; docker --version ; ps aux | fgrep -i docker; ls -al /var/run/containerd/containerd.sock

--- a/.github/workflows/ci_standalone_versioned.yml
+++ b/.github/workflows/ci_standalone_versioned.yml
@@ -55,7 +55,7 @@ jobs:
         run: sudo apt-get install -y libsasl2-dev python3-dev libldap2-dev libssl-dev build-essential
 
       - name: Install docker-compose
-        run: pip3 install --upgrade docker-compose
+        run: pip3 install --upgrade docker-py docker-compose
 
       - name: collect system info
         run: whoami; id; pwd; ls -al; uname -a ; df -h .; mount ; cat /etc/issue; docker --version ; ps aux | fgrep -i docker; ls -al /var/run/containerd/containerd.sock

